src/OrleansRuntime/Catalog/GrainCreator.cs

```diff
@@ -22,7 +25,16 @@ public GrainCreator(IGrainRuntime grainRuntime, IServiceProvider services = null
         {
             _grainRuntime = grainRuntime;
             _services = services;
```

Comment:
I see that during doing my PR that _services is stored in multiple places, instead of accessing it internally from current silo. But did not want to touch it for now. But if we're touching this part we should clean up and not store services instance in multiple places IMHO.